### PR TITLE
Flow rate unit is incomplete 131228425

### PR DESCRIPTION
**Pull request by thekcsam**
@opsmanager/developers , have a look on this? 😄 

To convert the mass flow rate (pound/time, kg/time) to volumetric flow rate (feet3/time, m3/time), I used this formula. 

mass flow rate x density of air = volumetric flow rate

I assumed the density of air to be 1.225kg/m3 for all scenario (actually this is only true in 15 celsius and 101.325kPa, see [here](http://engineering.stackexchange.com/questions/2561/converting-air-flow-rate-between-kg-s-and-m3-s) ),since the software seems like using this for different temperature and pressure as well.

### CODE REVIEW

File: spec/js/ko-components/view-models/orifice-calculator-spec.coffee [hunk 1]
```diff
@@ -182,17 +192,49 @@ define "orifice-calculator-viewmodel-spec", ["knockout", "jasmine-boot", "orific
...
+        viewModel.differentialPressure 30
+        viewModel.baseSpecificGravity 1
+        viewModel.operatingTemperature 60
+
+      it "should have default value of 'standard cubic feet'", ->
+        expect(viewModel.selectedFlowUnit()).toEqual config.FlowRatePressureUnits.standardCubicFeet
+
+      it "when standardCubicFeet is chosen", ->
+        viewModel.selectedFlowUnit config.FlowRatePressureUnits.standardCubicFeet
+        expect(viewModel.flowRate()).toEqual 32626.924 #32712
```
Comment (thekcsam) [inline spec/js/ko-components/view-models/orifice-calculator-spec.coffee:232]: The value in the comment is the value I got from the software.

File: src/js/ko-components/view-models/orifice-calculator.coffee [hunk 2]
```diff
@@ -104,12 +104,9 @@ define "orifice-calculator-viewmodel", ["knockout", "lodash", "knockout.validati
...
         @selectedCompressibilityCorrection() != config.CompressibilityCorrection.none
 
-      @compressibilityCorrectionValue = ko.observable()
-      @displayCompressibilityCorrection.subscribe (newValue) =>
-        if newValue
-          @compressibilityCorrectionValue 1
-        else
-          @compressibilityCorrectionValue undefined
+      @compressibilityCorrectionValue = ko.observable(1)
+      @displayCompressibilityCorrection.subscribe (isDisplayed) =>
```
Comment (thekcsam) [inline src/js/ko-components/view-models/orifice-calculator.coffee:110]: isDisplayed will return `true` or `false`.

File: src/js/lib/converter.js.coffee [hunk 3]
```diff
@@ -58,3 +59,13 @@ define "unit-converter", ["lodash"], (_) ->
...
     convert: (from, to, value) ->
       @CONSTANTS[from][to] * value
+
+  _.extend @OPL.Converter.Flowrate,
+    CONSTANTS:
+      "Standard Cubic Feet":
+        "Pounds": 0.0765
+        "Kilograms": 0.3048 ** 3 * 1.225
+        "Standard Cubic Meters": 0.3048 ** 3
+
```
Comment (thekcsam) [inline src/js/lib/converter.js.coffee:69]: Refer to [here](https://en.wikipedia.org/wiki/Density_of_air)

Density of air:
*At sea level and at 15 °C air has a density of approximately 1.225 kg/m3 (**0.001225 g/cm3**, 0.0023769 slug/ft3, **0.0765 lb/ft3**) according to ISA (International Standard Atmosphere)*

Hence,
standard cubic feet (ft3) * 0.0765 (lb/ft3) = lb
standard cubic feet (ft3) * 0.3048 (m/ft) ** 3 * 1.225 (kg/m3) = kg
standard cubic feet (ft3) * 0.3048 (m/ft) ** 3  = m3

File: spec/js/ko-components/views/orifice-calculator-spec.coffee [hunk 4]
```diff
@@ -57,5 +57,8 @@ define 'orifice-calculator-view-spec', ['jquery', 'jasmine-jquery'], ($) ->
       it 'should be in the template', ->
         expect($('#flow-rate')).toExist()
 
+    describe 'Flow Unit', ->
```
Comment (rpbaltazar) [inline spec/js/ko-components/views/orifice-calculator-spec.coffee]: Double quote on strings

File: src/js/lib/converter.js.coffee [hunk 5]
```diff
@@ -4,6 +4,7 @@ define "unit-converter", ["lodash"], (_) ->
   @OPL.Converter.Dimensions ||= {}
   @OPL.Converter.Temperature ||= {}
   @OPL.Converter.Pressure ||= {}
+  @OPL.Converter.Flowrate ||= {}
```
Comment (rpbaltazar) [inline src/js/lib/converter.js.coffee]: `FlowRate`

**Review by rpbaltazar — APPROVED** [review]
very little changes but overall good to merge

**Pull request merged**